lucille-examples/pom.xml

```diff
@@ -23,5 +23,6 @@
     <module>lucille-distributed-example</module>
     <module>lucille-simple-csv-solr-example</module>
     <module>lucille-opensearch-ingest-example</module>
+    <module>lucille-test-all-examples</module>
```

Comment:
Wondering if we could include TestAllExamples directly inside the examples module without having it be its own dedicated module underneath. (That's because it's not really an "example" meant to illustrate a lucille use case; rather, it's a test for all the actual examples.) Did you look into this by any chance?

Reply:
No, will look into it

Reply:
created test for each submodule of lucille-example instead in [cda4186](https://github.com/kmwtechnology/lucille/pull/196/commits/cda418650c577ca0310338503f1d95414107d749)